### DISCUSSION TITLE
using cli for transform jpg/png to ktx2 (basis, uastc)

### DISCUSSION THREAD

**Discussion by arpu** [donmccurdy/glTF-Transform#1495] [Q&A] [answered]
hi,

is there a sample script to use glt-tranform to transform  textures (jpg/png) to ktx2 with toktx? 

idea is to automatic transform textures to ktx2 normalmaps, baseColor,...

**Comment by donmccurdy** [comment, marked as answer]
Here's an example:

https://github.com/donmccurdy/glTF-Transform/blob/bdbecdd57eed8e1ccbf0388b0700dedb45fa6f08/packages/cli/src/cli.ts#L410-L435


The `toktx()` function is part of the /cli module rather than the /functions public API, because of #675, and it is possible that `toktx()` may have breaking changes outside of major version updates.
